packages/frontend-app/tailwind.config.js

```diff
@@ -34,6 +34,11 @@ module.exports = {
         telegram: "#85B6FF",
         linkedIn: "#0A66C2",
         facebook: "#0000FF",
+
+        // These are colors used in card component
+        title: "#ffffffeb",
+        artist: "#ffffffa3",
+        play: "#252526",
```

Comment:
Please remove these. The colors used for these texts are more or less secondary colors. 

Rebase your work, I added some new colors in the tailwind.config.js file you will find useful. One/some of the secondary colors should already do what you want.